sonarr/tag.go

```diff
@@ -91,8 +91,10 @@ func (s *Sonarr) DeleteTag(tagID int) error {
 }
 
 func (s *Sonarr) DeleteTagContext(ctx context.Context, tagID int) error {
+	var output interface{}
```

Comment:
I did this because this `Into()` methods close the response body and the others do not.

Reply:
Got it, it's fine for me.
Just a thought: Is there any case in which you map the `DELETE`'s response body to an actual struct? If not, maybe you can write another method instead of `DeleteInto`, just to avoid this.
Still, for me it's fine either way!

Reply:
There's always a chance someone comes along and requests that response from some delete. Today we do not provide it. I think I'm done with one-off methods though, and I'm fine with this approach.

And remember, the `APIer` interface can be used by consumers, so leaving `DeleteInto` allows others to figure it out.

Reply:
I think this may not work on some requests though; those with an empty response from a DELETE. Might have to do change the method anyway....

Reply:
```
[ERROR] 2022/09/04 21:17:34 Request failed. Status: 500: Internal Server Error, Message: deleting custom format: api.Delete(v3/customFormat): decoding json response body: EOF
```
I switched this up. Since Delete usually has no body.